cosmos/job/drm/drm_drmaa.py

```diff
@@ -46,21 +45,74 @@ def filter_is_done(self, tasks):
         jobid_to_task = {t.drm_jobID: t for t in tasks}
         # Keep yielding jobs until timeout > 1s occurs or there are no jobs
         while len(jobid_to_task):
+
+            failed_jobs = []
```

Comment:
i don't think there's a need for this list, just directly yield the failed task?